system-test/pubsub.js

```diff
@@ -92,6 +92,25 @@ describe('pubsub', function() {
     );
   }
 
+  function waitForTopics(done) {
+    async.retry(
+      {
+        times: 6,
+        interval: 10000,
+      },
+      function(callback) {
+        pubsub.getTopics(function(err, topics) {
+          if (!err && !topics.length) {
```

Comment:
I think we should test to make sure the topics that are returned match the topics that we created, so probably checking their names. If a cleanup cycle didn't execute from a previous run, this could be a false positive.

Reply:
PTAL!